tock/hours/tests/test_views.py

```diff
@@ -539,7 +528,8 @@ def setUp(self):
         )
 
     def test_user_reporting_period_report(self):
-        response = client(self).get(reverse(
```

Comment:
I'm not sure why this test was ever using `client`, as it's not testing an API view.  Honestly, I really think `client` should be imported as the name `api_client`--otherwise its use is quite ambiguous, as the containing file tests both standard website views _and_ API views.

Reply:
Thanks @toolness. Do you have any opinions on how the app would be rearchitected? Considering we've got you until the 19th on Tock, if you could jump in and make this easier to maintain that would be great. From an ATO perspective, I think it's fine to leave this all as-is, as it's not terrible to maintain right now. But I think I could use some help in the documentation for onboarding new devs. Specifically, explaining the current architecture and how it's a mix of different Django practices.

Reply:
Oh yeah sure, I'm happy to help with that!